contract_payment_mode/tests/test_contract_payment.py

```diff
@@ -4,42 +4,44 @@
 # Copyright 2017 Tecnativa - David Vidal
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo.tests import common
+import odoo.tests
 from ..hooks import post_init_hook
 
 
-class TestContractPaymentInit(common.SavepointCase):
-    @classmethod
-    def setUpClass(cls):
-        super(TestContractPaymentInit, cls).setUpClass()
+@odoo.tests.post_install(True)
+@odoo.tests.at_install(False)
+class TestContractPaymentInit(odoo.tests.HttpCase):
```

Comment:
I don't want to change to HttpCase, because it executes `setUp` for each test. I want `SavepointCase`

Reply:
I’m agree with you
My focus here is to get a travis green in order to search the real guilty

check my last comment from

https://github.com/OCA/maintainer-quality-tools/issues/509#issuecomment-343481818